### PR TITLE
Added Functionality to Enrich from a File of IPs

### PR DESCRIPTION
**Pull request by 0xTomTom**
Added a flag -f (file) to point to a file with 1 IP per line as input to enrich.

### CODE REVIEW

File: pkg/parser/main.go [hunk 1]
```diff
@@ -7,34 +7,56 @@ package parser
...
+}
+
 func (p *Parser) NewParser(file *os.File) *Parser {
 	return &Parser{
 		Decoder: json.NewDecoder(file),
 		File:    file,
 	}
 }
 
+func (p *Parser) ProcessSimpleScan() {
```
Comment (xstp) [inline pkg/parser/main.go:45]: can you have this return an error type?

File: pkg/parser/main.go [hunk 2]
```diff
@@ -68,6 +90,10 @@ func (p *Parser) EnrichScanRecords() {
 		uniqueIPAddresses[record.Ip] = struct{}{}
 	}
 
+	for index, ip := range uniqueIPAddresses {
```
Comment (xstp) [inline pkg/parser/main.go]: accidentally left behind?

**Comment by xstp** [comment]
Thanks for picking this up! Do you mind if I carry these over into my current PR?

File: pkg/parser/main.go (hunk 2 above)
Comment (0xTomTom) [reply pkg/parser/main.go]: Oeps ☹️

Comment (0xTomTom) [reply pkg/parser/main.go]: Removed this line

**Comment by 0xTomTom** [comment]
> Thanks for picking this up! Do you mind if I carry these over into my current PR?

Go for it!

**Pull request closed without merging**